setup_scripts/dagmc_setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#DAGMC Installation
+INSTALL_ROOT=$HOME
+cd $INSTALL_ROOT
+mkdir dagmc
+cd dagmc
+git clone https://github.com/svalinn/DAGMC
+cd DAGMC
+git checkout develop
+cd ..
+mkdir bld
+
+DAGMC_INSTALL_DIR=$INSTALL_ROOT/dagmc/install
+MOAB_CONFIG_DIR=$INSTALL_ROOT/MOAB/install/lib/cmake/MOAB
```

Comment:
It might be good to have a common file to source these locations from. `INSTALL_ROOT` here might not be the same as it is in the `moab_setup.sh` script. If one changes but not the other, there could be problems.

Reply:
How should I go about that? For instance, should I simply add an "install_root_location" file to the setup scripts dir, and have the others call their install roots from that?

Reply:
Something along those lines, yes. Maybe a file something like `install_locations.sh` and can be sourced (`source install_locations.sh`) in the various build script to keep the variables consistent.